src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -218,14 +218,15 @@ class WorkspacePeoplePage extends React.Component {
             .filter()
             .sortBy(person => person.displayName.toLowerCase())
             .value();
+        const policyID = lodashGet(this.props.route, 'params.policyID');
 
         return (
             <ScreenWrapper style={[styles.defaultModalContainer]}>
                 <HeaderWithCloseButton
-                    title={this.props.translate('common.people')}
+                    title={this.props.translate('common.members')}
                     onCloseButtonPress={() => Navigation.dismissModal()}
```

Comment:
```suggestion
                    onCloseButtonPress={Navigation.dismissModal}
```

Reply:
I don't want to change this one because `dismissModal()` takes an argument that I don't want to pass the event to.